packages/common/src/store/cache/collections/reducer.ts

```diff
@@ -98,54 +89,10 @@ const actionsMap = {
   }
 }
 
-const buildInitialState = (ssrPageProps?: SsrPageProps) => {
-  // If we have preloaded data from the server, populate the initial
-  // cache state with it
-  if (ssrPageProps?.collection) {
-    // @ts-ignore
-    const collection = makePlaylist(snakecaseKeys(ssrPageProps.collection))
-    if (!collection) return initialState
-
-    const id = collection.playlist_id
-    const uid = makeUid(Kind.COLLECTIONS, id)
-
-    return {
-      ...initialState,
-      entries: {
-        [id]: {
-          metadata: collection,
-          _timestamp: Date.now()
-        }
-      },
-      uids: {
-        [uid]: collection.playlist_id
-      },
-      statuses: {
-        [id]: 'SUCCESS'
-      },
-      permalinks: {
-        ...(collection.permalink
-          ? {
-              [collection.permalink]: collection.playlist_id
-            }
-          : {})
-      }
-    }
-  }
-  return initialState
```

Comment:
did this logic get moved somewhere else?

Reply:
Yeah we just set up the initial state in each onRenderHtml